apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/WriteTests.cs

```diff
@@ -131,6 +146,54 @@ public WriteTests(TestDatabaseFixture testFixture, ITestOutputHelper outputHelpe
         }
 
         [Fact]
+        public async Task WriteEmpties()
+        {
+            Assert.Equal(
+                1, await InsertAsync(
+                    new SpannerParameterCollection
+                    {
+                        new SpannerParameter(
+                            "BoolArrayValue",
+                            SpannerDbType.ArrayOf(SpannerDbType.Bool),
+                            new bool[] {}),
+                        new SpannerParameter(
+                            "Int64ArrayValue",
+                            SpannerDbType.ArrayOf(SpannerDbType.Int64),
+                            new long[] {}),
+                        new SpannerParameter(
+                            "Float64ArrayValue",
+                            SpannerDbType.ArrayOf(SpannerDbType.Float64),
+                            new double[] {}),
+                        new SpannerParameter(
+                            "StringArrayValue",
+                            SpannerDbType.ArrayOf(SpannerDbType.String),
+                            new string[] {}),
+                        new SpannerParameter(
+                            "BytesArrayValue",
+                            SpannerDbType.ArrayOf(SpannerDbType.Bytes),
+                            new byte[][] {}),
+                        new SpannerParameter(
+                            "TimestampArrayValue",
+                            SpannerDbType.ArrayOf(SpannerDbType.Timestamp),
+                            new DateTime[] {}),
+                        new SpannerParameter(
+                            "DateArrayValue",
+                            SpannerDbType.ArrayOf(SpannerDbType.Date),
+                            new DateTime[] {})
+                    }));
+            using (var reader = await GetLastRowAsync())
+            {
+                Assert.Equal(new bool[] {}, reader.GetFieldValue<bool[]>(reader.GetOrdinal("BoolArrayValue")));
```

Comment:
Just use `new bool[0]` etc instead of `new bool[] { }`? (Obviously not a bug, but slightly less wordy.)